sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageStreamSource.java

```diff
@@ -233,17 +233,17 @@ private synchronized boolean readNextRecord() throws IOException {
         fractionConsumedFromCurrentResponse = getFractionConsumed(currentResponse);
 
         Preconditions.checkArgument(
-            totalRowCountFromCurrentResponse > 0L,
-            "Row count from current response (%s) must be greater than one.",
+            totalRowCountFromCurrentResponse >= 0L,
+            "Row count from current response (%s) must be greater than or equal to one.",
```

Comment:
Seems like message does not match with the condition ? Do you just want to make sure that the row count is not negative for some reason ?

Reply:
And this is why we do code reviews! Done.